styles/octopus.css

```
@@ -0,0 +1,72 @@
+.octopus {
+    padding-bottom: 84px;
+
+    background: 
+    linear-gradient(252.43deg, #D45B78 0.01%, #4B3862 101.01%);
+}
+
+.octopus__container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    height: 100vh;
```

```suggestion
    height: calc(100vh - 84px);
```
You have a padding at the section, which must be taken into the calculation of the height, otherwise you get 100vh at the container + 84px at the section